sources/web/datalab/polymer/components/datalab-settings/datalab-settings.html

```diff
@@ -0,0 +1,44 @@
+<!--
+Copyright 2017 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+in compliance with the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License
+is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+or implied. See the License for the specific language governing permissions and limitations under
+the License.
+-->
+
+<link rel="import" href="../../components/shared-styles/shared-styles.html">
+
+<link rel="import" href="../../bower_components/paper-radio-group/paper-radio-group.html">
+<link rel="import" href="../../bower_components/paper-input/paper-input.html">
+
+<dom-module id="datalab-settings">
+  <template>
+    <style include="datalab-shared-styles">
+      paper-radio-button {
+        --paper-radio-button-checked-color: var(--selection-fg-color);
+      }
+      .setting {
+        padding: 15px;
+        border-bottom: 1px solid var(--border-color);
+      }
+    </style>
+
+    <!--Theme-->
+    <div class="setting">
+      <div>Theme:</div>
+      <paper-radio-group selected="{{theme}}" on-paper-radio-group-changed="_themeChanged">
```

Comment:
Minor nit- radio buttons seems like an odd choice here. It adds more visual clutter than a checkbox and doesn't scale as well as a dropdown menu.